mimic/model/clb_objects.py

```diff
@@ -14,6 +14,17 @@
                                                  _delete_node)
 
 
+class BadKeysError(Exception):
+    """
+    When trying to alter the settings of a load balancer, this exception will
+    be raised if you misspell or otherwise attempt to alter an attribute which
```

Comment:
minor nitpick: "misspell or otherwise" are redundant words here; simply saying "attempt to alter an attribute which doesn't exist" is pretty comprehensive documentation :)